docs/source/scientific_reference/1_time_series.rst

```diff
@@ -0,0 +1,165 @@
+* write :math:`s(\tau) := x_i` if :math:`\tau = t_i` and if
+there is a unique :math:`i` such that :math:`\tau = t_i`
+* call :math:`s(\tau)` the "value of :math:`s` at index :math:`\tau`"
+* may choose to say that :math:`s` is a time series if :math:`\mathcal{T}`
+is totally ordered and (order)-isomorphic to a sub-set of :math:`\mathbb{R}`,
+in a context where elements of :math:`\mathcal{T}` are interpreted as time stamps
+* write :math:`\mbox{series}(\mathcal{X}, \mathcal{T})` for the set of all index series
+with value domain :math:`\mathcal{X}`, index domain :math:`\mathcal{T}`
+
+Key operations for the above indexed series :math:`s` are:
+
+* inspecting :math:`\ell(s)`
+* inspecting :math:`\mbox{index}(s)` or :math:`\mbox{values}(s)`
+* for a :math:`\tau \in \mbox{index}(s)`, retrieve :math:`s(\tau)`
+* inspect the value domain :math:`\mathcal{X}` or index domain :math:`\mathcal{T}` of :math:`s`
+* sub-setting and selection operations
+
+For an indexed series :math:`s\in \mbox{series}(\mathcal{X}, \mathcal{T})`, we say that
+:math:`s` is:
+
+* univariate, if :math:`\mathcal{X}` is a basic primitive domain
+* multivariate, if :math:`\mathcal{X}` is a primitive domain but not a basic primitive domain
+* of primitive value domain, if :math:`\mathcal{X}` is a primitive domain; otherwise, of non-primitive value domain
+* degenerate, if :math:`\ell(s)=0` or :math:`\mathcal{X}` has one or less elements
+
+For a time series :math:`s\in \mbox{series}(\mathcal{X}, \mathcal{T})`, we say that
+:math:`s` is:
+
+* equally spaced, if :math:`t_{i+1} - t_i = c` for some :math:`c\in\mathbb{R}` and all
+    :math:`i\in 1,\dots, T-1`
+* unequally spaced, if :math:`s` is not equally spaced.
+
+We use the scitype `Series` to type univariate and multivariate time series.
+
+For `Series` typed,
+:math:`\mbox{series}(\mathcal{X}, \mathcal{T})`-valued random variables,
+we make no additional assumptions about the law (e.g., stationarity).
+
+
+Notes on the `Series` scitype
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``sktime`` conceptual model differs in some points from
+occasionally found conceptualization approaches in mathematical textbooks:
+
+* defining time series only over a finite index set :math:`t_1,\dots, t_T`,
+instead of, say, defining :math:`s(t)` for all :math:`t` in an infinite or even
+uncountably infinite set (e.g., the reals). This is because, in a computer,
+time series observations are always finite.
+Some books do not make this distinction and conflate "generative" knowledge
+in mathematical assumptions with observed information.
+We made this decisions since real observations, including those that ``python`` can handle, are always finite.
+* considering the index set as part of the time series and crucial, inspectable information.
+Some sources conceptualize the index set as "external" to the time series, or irrelevant.
+We think that considering the index set as part of the time series is conceptually natural for any user of ``pandas``.
```

Comment:
Yes, since ``pandas`` is so widely used that it is highly influential in formation of concepts.
What I want to say is that it is not the main *reason*, but users of ``pandas`` will be highly familiar with the concept - in contrast to people who read a lot of theory books but are not familiar with ``pandas`` (or R data frames).